var/spack/repos/builtin/packages/gcc/package.py

```diff
@@ -170,9 +211,9 @@ class Gcc(AutotoolsPackage, GNUMirrorPackage):
 
     # GCC 7.3 does not compile with newer releases on some platforms, see
     #   https://github.com/spack/spack/issues/6902#issuecomment-433030376
-    depends_on("mpfr@2.4.2:3.1.6", when="@:9.9")
-    depends_on("mpfr@3.1.0:", when="@10:")
-    depends_on("mpc@1.0.1:", when="@4.5:")
+    depends_on("mpfr@2.4.2:3.1.6")
+    depends_on("mpfr@3.1.0:")
+    depends_on("mpc@1.0.1:")
```

Comment:
Accidental change?

Reply:
Odd… yes, probably a rebase error or something? Will check, good spotting.

Reply:
Turns out this was not entirely a mistake.  There's an issue with a newer mpc or mpfr that causes gcc's configure to fail with mpc unable to find libm.so.6 despite having an rpath to it and a -L for the same path.  Using the older verison fixes it, so I'm going to restrict it for just the stages that need it and comment it.